codedoor/templates/codedoor/viewprofile.html

```diff
@@ -3,63 +3,61 @@
-                 {{ profile.user.get_full_name }}
-             </p>
-             <p id = "job">
-                 {{ profile.current_job }}
-             </p>
-             <p id = "classof">
-                 Class of {{ profile.graduation_year }}
-             </p>
-        </div>
-
-        <div id="info">
-            <!--{{ profile.website }}-->
-            <!--<p>-->
-                <!--<a href = "https://lausophia.github.io">-->
-                    <!--<img src= {% static 'images/linkicon.png' %} width="30px" height="30px">-->
-                    <!--lausophia.github.io-->
-                <!--</a>-->
-            <!--</p>-->
-            {% if profile.linkedin %}
-            <p>
-                <a href = {{ profile.linkedin }}>
-                    <img src={% static 'images/linkedinlogo.png' %} width="30px" height="30px">
-                    {{ profile.linkedin }}
-                </a>
-            </p>
-            {% endif %}
-            <!--{{profile.email}}-->
-            <p>
-                <a href = "mailto:{{ profile.user.email }}">
-                    <img src={% static 'images/emailicon.png' %} width="30px" height="30px">
-                    {{ profile.user.email }}
-                </a>
-            </p>
-            <!--<a href={{ profile.resume }}>-->
-            <!--<p>-->
-                <!--<img src={% static 'images/cvicon.png' %} width="30px" height="30px">-->
-                <!--resume.pdf &lt;!&ndash;</a>&ndash;&gt;-->
-            <!--</p>-->
-            {% if request.user.profile == profile %}
-        <p id = "edit-profile">
-            <a href="/codedoor/editprofile/{{ profile.id }}">
-            (edit profile)
-            </a>
+    <div class="profile-information">
+      <div class="basic-info">
+        <p id="name">
+          {{ profile.user.get_full_name }}
+        </p>
+        <p id="job">
+          {{ profile.current_job }}
         </p>
-    {% endif %}
+        <p id="classof">
+          Class of {{ profile.graduation_year }}
+        </p>
+      </div>
+      <div id="profile-links">
+        <!--{{ profile.website }}-->
+        <!--<p>-->
+            <!--<a href = "https://lausophia.github.io">-->
+                <!--<img src= {% static 'images/linkicon.png' %} width="30px" height="30px">-->
+                <!--lausophia.github.io-->
+            <!--</a>-->
+        <!--</p>-->
+        {% if profile.linkedin %}
+          <div class="link">
+            <a href={{ profile.linkedin }}>
+              <img class="icon" src={% static 'images/linkedinlogo.png' %}>
+              <span>{{ profile.linkedin }}</span>
+            </a>
+          </div>
+        {% endif %}
+        <div class="link">
+          <a href="mailto:{{ profile.user.email }}">
+            <img class="icon" src={% static 'images/emailicon.png' %}>
+            <span>{{ profile.user.email }}</span>
+          </a>
         </div>
+        <!--<a href={{ profile.resume }}>-->
```

Comment:
We can remove this for now because resumes aren't supported yet (though we have a resume field in the schema). Adding this feature isn't a priority at the moment.